
package com.sforce.partner;

import javax.xml.ws.WebFault;


/**
 * This class was generated by Apache CXF 3.3.7
 * 2020-09-21T07:44:16.938-05:00
 * Generated source version: 3.3.7
 */

@WebFault(name = "UnexpectedErrorFault", targetNamespace = "urn:fault.partner.soap.sforce.com")
public class UnexpectedErrorFault_Exception extends Exception {

    private com.sforce.partner.UnexpectedErrorFault unexpectedErrorFault;

    public UnexpectedErrorFault_Exception() {
        super();
    }

    public UnexpectedErrorFault_Exception(String message) {
        super(message);
    }

    public UnexpectedErrorFault_Exception(String message, java.lang.Throwable cause) {
        super(message, cause);
    }

    public UnexpectedErrorFault_Exception(String message, com.sforce.partner.UnexpectedErrorFault unexpectedErrorFault) {
        super(message);
        this.unexpectedErrorFault = unexpectedErrorFault;
    }

    public UnexpectedErrorFault_Exception(String message, com.sforce.partner.UnexpectedErrorFault unexpectedErrorFault, java.lang.Throwable cause) {
        super(message, cause);
        this.unexpectedErrorFault = unexpectedErrorFault;
    }

    public com.sforce.partner.UnexpectedErrorFault getFaultInfo() {
        return this.unexpectedErrorFault;
    }
}
